nixos/doc/manual/default.nix

```diff
@@ -72,6 +73,63 @@ let
       </toc>
     '';
 
+  manualXsltprocOptions = toString [
```

Comment:
I'll revert the commit here and open it as a new PR separately. The build of the manual won't fail if this PR is merged but it will issue an ugly warning about unresolved olinks. The manpage will contain something like `More instructions about NixOS in conjuction with Taskserver can be found in the NixOS manual at ???`, so I guess it shouldn't do much harm except for the missing reference in the Taskserver option description.